src/main.rs

```diff
@@ -693,12 +693,15 @@ fn main() {
 
           dirty = true;
 
+          let ahead_behind = format!("[ahead {}, behind {}]",
```

Comment:
IMO the ahead/behind text should only be there if we're actually ahead and/or behind: a repo with just dirty changes shouldn't have this text show up.